apps/standalone/deploy/helm/flightctl-ui/values.yaml

```
@@ -3,6 +3,8 @@ flightctl-ui:
   image: quay.io/flightctl/flightctl-ui:latest
   flightctlServer: https://flightctl-api:3443
   flightctlMetricsServer: https://flightctl-api:9090
+  bootc-img-url: quay.io/flightctl/flightctl-agent-centos:bootstrap
+  qcow2-img-url: https://fctl-ui-img.s3.amazonaws.com/disk.qcow2
```

as these are specific for each deployment. I'd rather left them out here and just use some placeholder as we do for other values. Something like 
```suggestion
  bootc-img-url: quay.io/example/example-agent-centos
  qcow2-img-url: https://example.com/disk.qcow2
```